Formula/websocat.rb

```diff
@@ -0,0 +1,24 @@
+class Websocat < Formula
+  desc "Command-line client for WebSockets"
+  homepage "https://github.com/vi/websocat"
+  url "https://github.com/vi/websocat/archive/v1.2.0.tar.gz"
+  sha256 "e1a6021770c9ae5b364eaa74b7435f259c5d84bf5c7210cab7f0c775fa2158a5"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+
+  # This formula is based on ripgrep.rb
+  def install
+    ENV["XML_CATALOG_FILES"] = etc/"xml/catalog"
```

Comment:
I think you don't need this.
(In the `ripgrep` formula, this is required for `docbook-xsl` to work correctly)